codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/NestedAccessorGenerator.kt

```diff
@@ -24,7 +22,7 @@ import software.amazon.smithy.rust.codegen.core.smithy.protocols.lensName
 
 /** Generator for accessing nested fields through optional values **/
 class NestedAccessorGenerator(private val symbolProvider: RustSymbolProvider) {
-    private val module = RustModule("lens", RustMetadata(visibility = Visibility.PUBLIC), "Generated accessors for nested fields")
+    private val module = RustModule.private("lens", "Generated accessors for nested fields")
```

Comment:
I'm wondering if we should make documentation a required field for public modules but remove it from the private module creation fn. I like having docs everywhere but I know we have a rule of not including them unless they'll publicly visible.

Reply:
> I like having docs everywhere but I know we have a rule of not including them unless they'll publicly visible.

It's more of a leniency than a rule. We can have docs for private items. It's just not a requirement like it is for public items (although we still need to enable the lint for this in some of the rust-runtime crates).